### PR TITLE
Add sqlite3 to Gemfile and remove from Appraisals

### LINKED ISSUE #884: Unable to do intial setup locally
I forked & pull the gem, trying to [setup](https://github.com/udaykadaboina/clearance/blob/master/CONTRIBUTING.md) clearance after downloading.

`.bin/setup` command gives following error:
The Gemfile's dependencies are satisfied
>> bundle check --gemfile='/Users/uday/Development/clearance/gemfiles/rails_5.0.gemfile' || bundle install --gemfile='/Users/uday/Development/clearance/gemfiles/rails_5.0.gemfile' --retry 1
The Gemfile's dependencies are satisfied
>> bundle check --gemfile='/Users/uday/Development/clearance/gemfiles/rails_5.1.gemfile' || bundle install --gemfile='/Users/uday/Development/clearance/gemfiles/rails_5.1.gemfile' --retry 1
The Gemfile's dependencies are satisfied
>> bundle check --gemfile='/Users/uday/Development/clearance/gemfiles/rails_5.2.gemfile' || bundle install --gemfile='/Users/uday/Development/clearance/gemfiles/rails_5.2.gemfile' --retry 1
The Gemfile's dependencies are satisfied
>> bundle check --gemfile='/Users/uday/Development/clearance/gemfiles/rails_6.0.gemfile' || bundle install --gemfile='/Users/uday/Development/clearance/gemfiles/rails_6.0.gemfile' --retry 1
The Gemfile's dependencies are satisfied
rake aborted!
LoadError: Error loading the 'sqlite3' Active Record adapter. Missing a gem it depends on? sqlite3 is not part of the bundle. Add it to your Gemfile.
/Users/uday/.rvm/gems/ruby-2.6.3@clearance/gems/bundler-2.1.4/lib/bundler/rubygems_integration.rb:346:in `block (2 levels) in replace_gem'
/Users/uday/.rvm/gems/ruby-2.6.3@clearance/gems/activerecord-6.0.2.1/lib/active_record/connection_adapters/sqlite3_adapter.rb:13:in `<top (required)>'
/Users/uday/.rvm/gems/ruby-2.6.3@clearance/gems/activesupport-6.0.2.1/lib/active_support/dependencies.rb:325:in `require'
/Users/uday/.rvm/gems/ruby-2.6.3@clearance/gems/activesupport-6.0.2.1/lib/active_support/dependencies.rb:325:in `block in require'
/Users/uday/.rvm/gems/ruby-2.6.3@clearance/gems/activesupport-6.0.2.1/lib/active_support/dependencies.rb:291:in `load_dependency'
/Users/uday/.rvm/gems/ruby-2.6.3@clearance/gems/activesupport-6.0.2.1/lib/active_support/dependencies.rb:325:in `require'
/Users/uday/.rvm/gems/ruby-2.6.3@clearance/gems/activerecord-6.0.2.1/lib/active_record/connection_adapters/connection_specification.rb:170:in `spec'
/Users/uday/.rvm/gems/ruby-2.6.3@clearance/gems/activerecord-6.0.2.1/lib/active_record/connection_adapters/abstract/connection_pool.rb:1054:in `establish_connection'
/Users/uday/.rvm/gems/ruby-2.6.3@clearance/gems/activerecord-6.0.2.1/lib/active_record/connection_handling.rb:51:in `establish_connection'
/Users/uday/.rvm/gems/ruby-2.6.3@clearance/gems/activerecord-6.0.2.1/lib/active_record/railtie.rb:201:in `block (2 levels) in <class:Railtie>'
/Users/uday/.rvm/gems/ruby-2.6.3@clearance/gems/activesupport-6.0.2.1/lib/active_support/lazy_load_hooks.rb:72:in `class_eval'
/Users/uday/.rvm/gems/ruby-2.6.3@clearance/gems/activesupport-6.0.2.1/lib/active_support/lazy_load_hooks.rb:72:in `block in execute_hook'
/Users/uday/.rvm/gems/ruby-2.6.3@clearance/gems/activesupport-6.0.2.1/lib/active_support/lazy_load_hooks.rb:62:in `with_execution_control'
/Users/uday/.rvm/gems/ruby-2.6.3@clearance/gems/activesupport-6.0.2.1/lib/active_support/lazy_load_hooks.rb:67:in `execute_hook'
/Users/uday/.rvm/gems/ruby-2.6.3@clearance/gems/activesupport-6.0.2.1/lib/active_support/lazy_load_hooks.rb:52:in `block in run_load_hooks'
/Users/uday/.rvm/gems/ruby-2.6.3@clearance/gems/activesupport-6.0.2.1/lib/active_support/lazy_load_hooks.rb:51:in `each'
/Users/uday/.rvm/gems/ruby-2.6.3@clearance/gems/activesupport-6.0.2.1/lib/active_support/lazy_load_hooks.rb:51:in `run_load_hooks'
/Users/uday/.rvm/gems/ruby-2.6.3@clearance/gems/activerecord-6.0.2.1/lib/active_record/base.rb:327:in `<module:ActiveRecord>'
/Users/uday/.rvm/gems/ruby-2.6.3@clearance/gems/activerecord-6.0.2.1/lib/active_record/base.rb:27:in `<top (required)>'
/Users/uday/.rvm/gems/ruby-2.6.3@clearance/gems/activesupport-6.0.2.1/lib/active_support/dependencies.rb:325:in `require'
/Users/uday/.rvm/gems/ruby-2.6.3@clearance/gems/activesupport-6.0.2.1/lib/active_support/dependencies.rb:325:in `block in require'
/Users/uday/.rvm/gems/ruby-2.6.3@clearance/gems/activesupport-6.0.2.1/lib/active_support/dependencies.rb:291:in `load_dependency'
/Users/uday/.rvm/gems/ruby-2.6.3@clearance/gems/activesupport-6.0.2.1/lib/active_support/dependencies.rb:325:in `require'
/Users/uday/.rvm/gems/ruby-2.6.3@clearance/gems/activerecord-6.0.2.1/lib/active_record/railties/databases.rake:19:in `block (2 levels) in <top (required)>'
/Users/uday/.rvm/gems/ruby-2.6.3@clearance/gems/rake-13.0.1/exe/rake:27:in `<top (required)>'
/Users/uday/.rvm/gems/ruby-2.6.3@clearance/bin/ruby_executable_hooks:24:in `eval'
/Users/uday/.rvm/gems/ruby-2.6.3@clearance/bin/ruby_executable_hooks:24:in `<main>'

Caused by:
Gem::LoadError: sqlite3 is not part of the bundle. Add it to your Gemfile.
/Users/uday/.rvm/gems/ruby-2.6.3@clearance/gems/bundler-2.1.4/lib/bundler/rubygems_integration.rb:346:in `block (2 levels) in replace_gem'
/Users/uday/.rvm/gems/ruby-2.6.3@clearance/gems/activerecord-6.0.2.1/lib/active_record/connection_adapters/sqlite3_adapter.rb:13:in `<top (required)>'
/Users/uday/.rvm/gems/ruby-2.6.3@clearance/gems/activesupport-6.0.2.1/lib/active_support/dependencies.rb:325:in `require'
/Users/uday/.rvm/gems/ruby-2.6.3@clearance/gems/activesupport-6.0.2.1/lib/active_support/dependencies.rb:325:in `block in require'
/Users/uday/.rvm/gems/ruby-2.6.3@clearance/gems/activesupport-6.0.2.1/lib/active_support/dependencies.rb:291:in `load_dependency'
/Users/uday/.rvm/gems/ruby-2.6.3@clearance/gems/activesupport-6.0.2.1/lib/active_support/dependencies.rb:325:in `require'
/Users/uday/.rvm/gems/ruby-2.6.3@clearance/gems/activerecord-6.0.2.1/lib/active_record/connection_adapters/connection_specification.rb:170:in `spec'
/Users/uday/.rvm/gems/ruby-2.6.3@clearance/gems/activerecord-6.0.2.1/lib/active_record/connection_adapters/abstract/connection_pool.rb:1054:in `establish_connection'
/Users/uday/.rvm/gems/ruby-2.6.3@clearance/gems/activerecord-6.0.2.1/lib/active_record/connection_handling.rb:51:in `establish_connection'
/Users/uday/.rvm/gems/ruby-2.6.3@clearance/gems/activerecord-6.0.2.1/lib/active_record/railtie.rb:201:in `block (2 levels) in <class:Railtie>'
/Users/uday/.rvm/gems/ruby-2.6.3@clearance/gems/activesupport-6.0.2.1/lib/active_support/lazy_load_hooks.rb:72:in `class_eval'
/Users/uday/.rvm/gems/ruby-2.6.3@clearance/gems/activesupport-6.0.2.1/lib/active_support/lazy_load_hooks.rb:72:in `block in execute_hook'
/Users/uday/.rvm/gems/ruby-2.6.3@clearance/gems/activesupport-6.0.2.1/lib/active_support/lazy_load_hooks.rb:62:in `with_execution_control'
/Users/uday/.rvm/gems/ruby-2.6.3@clearance/gems/activesupport-6.0.2.1/lib/active_support/lazy_load_hooks.rb:67:in `execute_hook'
/Users/uday/.rvm/gems/ruby-2.6.3@clearance/gems/activesupport-6.0.2.1/lib/active_support/lazy_load_hooks.rb:52:in `block in run_load_hooks'
/Users/uday/.rvm/gems/ruby-2.6.3@clearance/gems/activesupport-6.0.2.1/lib/active_support/lazy_load_hooks.rb:51:in `each'
/Users/uday/.rvm/gems/ruby-2.6.3@clearance/gems/activesupport-6.0.2.1/lib/active_support/lazy_load_hooks.rb:51:in `run_load_hooks'
/Users/uday/.rvm/gems/ruby-2.6.3@clearance/gems/activerecord-6.0.2.1/lib/active_record/base.rb:327:in `<module:ActiveRecord>'
/Users/uday/.rvm/gems/ruby-2.6.3@clearance/gems/activerecord-6.0.2.1/lib/active_record/base.rb:27:in `<top (required)>'
/Users/uday/.rvm/gems/ruby-2.6.3@clearance/gems/activesupport-6.0.2.1/lib/active_support/dependencies.rb:325:in `require'
/Users/uday/.rvm/gems/ruby-2.6.3@clearance/gems/activesupport-6.0.2.1/lib/active_support/dependencies.rb:325:in `block in require'
/Users/uday/.rvm/gems/ruby-2.6.3@clearance/gems/activesupport-6.0.2.1/lib/active_support/dependencies.rb:291:in `load_dependency'
/Users/uday/.rvm/gems/ruby-2.6.3@clearance/gems/activesupport-6.0.2.1/lib/active_support/dependencies.rb:325:in `require'
/Users/uday/.rvm/gems/ruby-2.6.3@clearance/gems/activerecord-6.0.2.1/lib/active_record/railties/databases.rake:19:in `block (2 levels) in <top (required)>'
/Users/uday/.rvm/gems/ruby-2.6.3@clearance/gems/rake-13.0.1/exe/rake:27:in `<top (required)>'
/Users/uday/.rvm/gems/ruby-2.6.3@clearance/bin/ruby_executable_hooks:24:in `eval'
/Users/uday/.rvm/gems/ruby-2.6.3@clearance/bin/ruby_executable_hooks:24:in `<main>'
Tasks: TOP => dummy:db:reset => dummy:db:drop => dummy:db:load_config
(See full trace by running task with --trace)

### PR DESCRIPTION
**Pull request by gnfisher**
Closes #884.

Before this commit, running the `bin/setup` script was breaking. This was
because `bundle exec rake dummy:db:reset` command used the top level Gemfile
where the sqlite3 gem was not specified. It was only specified in the Appraisal
Gemfile.

It was necessary to specify different sqlite3 versions when we were running the
test suite against both Rails 4.X and Rails 6.X. We no longer support Rails 4.X
and so it was no longer needed.

This commit specifies the sqlite3 gem in the top level Gemfile and removes the
specification from the `Appraisal` file.

Co-Authored-By: Daniel Colson <danieljamescolson@gmail.com>
Co-Authored-By: Eebs Kobeissi <ebrahim.kobeissi@gmail.com>

### CODE REVIEW

**Comment by composerinteralia** [comment]
Oops, it looks like Rails 5 also had the `~> 1.3.6` constraint on sqlite. See https://github.com/thoughtbot/factory_bot/blob/master/Appraisals for once possible way of addressing this.

**Comment by gnfisher** [comment]
> Oops, it looks like Rails 5 also had the `~> 1.3.6` constraint on sqlite. See https://github.com/thoughtbot/factory_bot/blob/master/Appraisals for once possible way of addressing this. On factory_bot we do also specify the gem in the top level Gemfile: https://github.com/thoughtbot/factory_bot/blob/master/Gemfile

Thanks. It looks like factory_bot includes `gem sqlite3` in the top-level file as well as specifying  versions in Appraisals.

File: NEWS.md [hunk 1]
```diff
@@ -6,7 +6,8 @@ complete changelog, see the git history for each version via the version links.
 ### Fixed
 
 - Delete cookie correctly when a callable object is set as the custom domain
-  setting.
```
Comment (eebs) [inline NEWS.md:9]: Oops, looks like this line was accidentally removed.

**Comment by composerinteralia** [comment]
> We may need to set something more restrictive in the gemfiles/xxx.gemfile files:

We should set that in the Appraisal file rather then the auto-generated gemfiles. Otherwise what is in the gemfile would get updated back to the latest version of capybara when we run `appraisal update`.

**Comment by eebs** [comment]
> We should set that in the Appraisal file rather then the auto-generated gemfiles. Otherwise what is in the gemfile would get updated back to the latest version of capybara when we run `appraisal update`.

Oh, neat! I didn't know those were generated, I don't know how Appraisal works. I'll go read up on it 😁

**Comment by composerinteralia** [comment]
You can ignore my last comment for now if the tests pass. I want make a change to the appraisal file to make it easier to update gem versions for specific Rails version without having to write conditionals and I can move where we specifiy the capybara version then.

**Pull request merged**